### PR TITLE
Update inbound-sms.md

### PR DESCRIPTION
**Pull request by brianebert**
Added 204 response option

## Description

Please describe what the changes are made in this pull request and why the change was necessary.

## Deploy Notes

Notes regarding deployment the contained body of work. These should note any db migrations, etc.

### CODE REVIEW

File: _documentation/en/messaging/sms/guides/inbound-sms.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ navigation_weight: 5
 
 To receive inbound SMS, create a [webhook endpoint](/concepts/guides/webhooks) and configure your account to use it in the API settings tab of the [Nexmo Developer Dashboard](https://dashboard.nexmo.com/settings).
 
-Any messages sent to one of your Nexmo virtual numbers are sent to your webhook using a [GET] or [POST] request. When you receive a message on your webhook, you must send a `200 OK` response. If you don't, then Nexmo will assume that you have not received the message and will keep resending it for the next 24 hours.
+Any messages sent to one of your Nexmo virtual numbers are sent to your webhook using a [GET] or [POST] request. When you receive a message on your webhook, you must send a `200 OK` or '204 No Content' response. If you don't, then Nexmo will assume that you have not received the message and will keep resending it for the next 24 hours.
```
Comment (mheap) [inline _documentation/en/messaging/sms/guides/inbound-sms.md:11]: Could you replace the quotes with backticks to indicate that this is a literal response string?

**Review by mheap — CHANGES_REQUESTED** [review]

**Comment by lukeocodes** [comment]
@brianebert please make the changes or I'll be forced to close this PR

**Comment by brianebert** [comment]
Mr. Olif,

Are you saying that if I don't change quotation marks to back ticks, or whatever you asked for, that you intend to close my pr without correcting the substance of your documentation error?

Go ahead and close my pr. I don't really care. You are a for profit organization communicating with a volunteer who noticed a problem and tried to help other nexmo users. I await an answer to my question. Please cc your manager.

BE

Get your free encrypted mail account at protonmail.com and your communications with Motia will be encrypted end to end.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by mheap** [comment]
Hi Brian,

Thanks for your interest in improving our docs. Your current proposed change would result in our tooling trying to translate `204 No Contact` as it's not designated as a literal string.

In line with your request, I'm closing your PR and will address the inconsistency that you've identified personally to make sure that our audience has all the information.

Cheers, Michael

**Pull request closed without merging**